### ISSUE TITLE
Provide a dedicated page with GitHub app commands

### ISSUE THREAD

**Issue by TomasTomecek** [packit/packit.dev#573]
`/packit help` is a solution but clutters PR

We should instead have a dedicated page where all commands are listed and documented.

**Comment by lachmanfrantisek** [comment]
Maybe just add a new comment with a link to the doc page? (So we don't need to have that info in the packit-service code.)

**Comment by lachmanfrantisek** [comment]
> We should instead have a dedicated page where all commands are listed and documented.

Do we want to have this for all the jobs in one place? Or document this for each job?

**Comment by TomasTomecek** [comment]
The ask was that it's hard to find all the supported `/packit` commands as they are spread all over our documentation. So ideally a single page or group of pages per command.

**Comment by lachmanfrantisek** [comment]
I would say we already have something like that:
https://packit.dev/docs/guide/#how-to-re-trigger-packit-actions-in-your-pull-request

To improve that, we can generalise the text/title (= include issue comments) and link it from other places.

**Comment by TomasTomecek** [comment]
Yes, true, but there is also propose_downstream, verify-fas, vm-image-build... :)

**Comment by mfocko** [comment]
@mfocko to check how related it is to the page with commands for retriggering
